content/docs/latest/spire-helm-charts-hardened-about/recommendations.md

```diff
@@ -0,0 +1,94 @@
+across the two recommended namespaces for the services:
+
+| Namespace Type | Namespace Value                     | Default Value | Purpose                                                    |
+| -------------- | ----------------------------------- | ------------- | ---------------------------------------------------------- |
+| Server         | global.spire.namespaces.server.name | spire-server  | Services that should have restricted Kubernetes privileges |
+| System         | global.spire.namespaces.system.name | spire-system  | Services needing Kubernetes privileges                     |
+
+| Service Name                   | Namespace Type |
+| ------------------------------ | -------------- |
+| SPIRE Server                   | Server         |
+| SPIFFE OIDC Discovery Provider | Server         |
+| SPIFFE CSI Driver              | System         |
+| SPIRE Agent                    | System         |
+
+## Namespace Pod Security Standards
+
+Option `global.spire.recommendations.namespacePSS` sets the chart to set the recommended
+[Kubernetes Pod Security Standard](https://kubernetes.io/docs/concepts/security/pod-security-standards/) labels when namespaces
+are created with the chart via:
+
+| Value                                 | Default Value | Description                                                                                 |
+| ------------------------------------- | ------------- | ------------------------------------------------------------------------------------------- |
+| global.spire.namespaces.create        | false         | Create both recommeded namespaces                                                           |
+| global.spire.namespaces.spire.create  | false         | Create the namespace specified by global.spire.namespaces.spire.name (default spire-server) |
+| global.spire.namespaces.system.create | false         | Create the namespace specified by global.spire.namespaces.spire.name (default spire-server) |
+
+On creation, the following Namespaces are assigned their Pod Security Standard:
+
+| Namespace Type | Pod Secuirty Standard |
+| -------------- | --------------------- |
+| Server         | Restricted            |
+| System         | Privileged            |
+
+
+## Priority Class Name
+
+Option `global.spire.recommendations.priorityClassName` sets the Kubernetes Priority Class Names so that if there is resource contention on the cluster
+the SPIRE Services will have a very high priority. SPIRE malfunctioning can cuase other important workloads to malfunction too so we prevent that from
+happening.
+
+## Prometheus
+
+Option `global.spire.recommendations.prometheus` enables prometheus style exporters to be exposed out of the relvant pods. This enables Prometheus or
+other compatable services to gather metrics from the various services.
+
+## Strict Mode
+
+Option `global.spire.recommendations.strictMode` adds additional checks on the configuration to help ensure your configuration is production ready.
```

Comment:
Is there somewhere readers can learn more about what these additional checks are?

Reply:
It mostly makes sure they explicitly set stuff like, the trust domain, cluster name, and ca related settings. Basically all of the settings that were mentioned in the production installation instructions. Would it be helpful to try and document it here and keep it in sync with the chart, or we could refer back to the install instructions?

Reply:
Updated